scripts/data/common/killed-in-gaza/data/dict_ar_ar.csv

```diff
@@ -1,127 +1,127 @@
-original,cleaned
-عبد الله,عبدالله
-ابو ,ابو
-بهاء الدين,بهاءالدين
-جاد الله,جادالله
-حرز الله,حرزالله
-ساق الله,ساقالله
-صلاح الدين,صلاحالدين
-عبد الجواد,عبدالجواد
-عبد الحفيظ,عبدالحفيظ
-عبد الحكيم,عبدالحكيم
-عبد الحليم,عبدالحليم
-عبد الحميد,عبدالحميد
-عبد الخالق,عبدالخالق
-عبد الرؤوف,عبدالرؤوف
-عبد الرحمن,عبدالرحمن
-عبد الرحيم,عبدالرحيم
-عبد الرحيم,عبدالرحيم
-عبد العاطي,عبدالعاطي
-عبد العال,عبدالعال
-عبد العزيز,عبدالعزيز
-عبد العظيم,عبدالعظيم
-عبد الفتاح,عبدالفتاح
-عبد الفتاح,عبدالفتاح
-عبد القادر,عبدالقادر
-عبد الكريم,عبدالكريم
-عبد اللطيف,عبداللطيف
-عبد المالك,عبدالمالك
-عبد المحسن,عبدالمحسن
-عبد الناصر,عبدالناصر
-عبد النبي,عبدالنبي
-عبد الهادي,عبدالهادي
-عبد ربه,عبدربه
-عز الدين,عزالدين
-عطا الله,عطاالله
-عوض الله,عوضالله
-مجد الدين,مجدالدين
-محي الدين,محيالدين
-منة الله,منةالله
-نور الدين,نورالدين
-عبد المعطي,عبدالمعطي
-عبد السلام,عبدالسلام
-عبد الرزاق,عبدالرزاق
-عبد الغفور,عبدالغفور
-منه الله,منهالله
-عماد الدين,عمادالدين
-نصر الدين,نصرالدين
-نور الهدى,نورالهدى
-عبد الوهاب,عبدالوهاب
-عبد الرازق,عبدالرازق
-عبد الساتر,عبدالساتر
-عبد ربو,عبدربو
-عبد المجيد,عبدالمجيد
-عبد الحافظ,عبدالحافظ
-عبد النور,عبدالنور
-علاء الدين,علاءالدين
-عبد الواحد,عبدالواحد
-عبد الجابر,عبدالجابر
-عبد المنعم,عبدالمنعم
-عبد الجليل,عبدالجليل
-عبد الباري,عبدالباري
-سيف الدين,سيفالدين
-خليل الرحمن,خليلالرحمن
-زين الدين,زينالدين
-سراج الدين,سراجالدين
-حسام الدين,حسامالدين
-غلام الدين,غلامالدين
-عبد المولى,عبدالمولى
-غلام الدين,غلامالدين
-عبد دبور,عبددبور
-عوده الله,عودهالله
-جمال الدين,جمالالدين
-عبد الحي,عبدالحي
-فاطمه الزهراء,فاطمهالزهراء
-ضياء الدين,ضياءالدين
-عبد الغني,عبدالغني
-زين العابدين,زينالعابدين
-عبد الستار,عبدالستار
-عبد الحق,عبدالحق
-عبد الدايم,عبدالدايم
-سعد الدين,سعدالدين
-ضيف الله,ضيفالله
-عبد الحق,عبدالحق
-فرج الله,فرجالله
-ضياء الحق,ضياءالحق
-هبة الله,هبةالله
-عيد الزقزوق,عيدالزقزوق
-عبد الغفار,عبدالغفار
-جاد الحق,جادالحق
-عبد الصمد,عبدالصمد
-عبد السميع,عبدالسميع
-سيف الاسلام,سيفالاسلام
-ناصر الدين,ناصرالدين
-فاطمة الزهراء,فاطمةالزهراء
-عبد الباقي,عبدالباقي
-عبد الشافي,عبدالشافي
-عودة الله,عودةالله
-آية الله,آيةالله
-سعد الله,سعدالله
-عبد الجبار,عبدالجبار
-ابو الليل,ابوالليل
-عبد العزيز,عبدالعزيز
-عبد الكريم,عبدالكريم
-تيم الله,تيمالله
-حسب الله,حسبالله
-صنع الله,صنعالله
-خلف الله,خلفالله
-سيف الاسلام,سيفالاسلام
- عيد دردونه, عيددردونه
-عودت الله,عودتالله
-عبد الحسني,عبدالحسني
-عبد السطري,عبدالسطري
-عبد الدردساوي,عبدالدردساوي
-عبد حجاج,عبدحجاج
-عبد مقاط,عبدمقاط
-عبد القرم,عبدالقرم
-عبد فتوح,عبدفتوح
-عبد المظلوم,عبدالمظلوم
-عبد سليم,عبدسليم
-عبد دلول,عبددلول
-عبد سليمان,عبدسليمان
-عبد محمود,عبدمحمود
-عبد خليل,عبدخليل
-عبد عوض,عبدعوض
-عبد القرا,عبدالقرا
-عبد اللحام,عبداللحام
-عبد العقاد,عبدالعقاد
+original,cleaned
```

Comment:
this file was sorted by the leftside value and normalized using the library noted in the description which removes diacritics, tatweel and superscript alef

spaces are also trimmed from the beginning / end of the leftside value if they exist

Reply:
I'll bring back the spaces handling you mentioned was intentional in discord

The library is here to improve lookup reliability and spotting duplicates, not to replace the list which it can't